spec/features/admin_adds_new_video.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+feature 'Admin adds new video' do
+  scenario 'Admin succesfully adds a new video' do
+    admin = Fabricate(:admin)
+    dramas = Fabricate(:category, name: "Dramas")
+    sign_in(admin)
+    visit new_admin_video_path
+
+    fill_in "Title", with: "Monk"
+    select "Dramas", from: "Category"
+    fill_in "Description", with: "SF detective"
+    attach_file "Large Cover", "spec/support/uploads/monk_large.jpg"
+    attach_file "Small Cover", "spec/support/uploads/monk.jpg"
+    fill_in "Video URL", with: "http://www.example.com/my_video.mp4"
+    click_button "Add Video"
+
+    sign_out
+    sign_in
+
+    visit video_path(Video.first)
+    expect(page).to have_selector("img[src='/uploads/monk_large.jpg']")
+    expect(page).to have_selector("a[href='http://www.example.com/my_video.mp4']")
```

Comment:
It would be nice to have these last 2 expectations extracted to some nice readable methods. To keep things aligned.